gui/shredder/runner.py

```diff
@@ -57,8 +57,8 @@ class MatchType(Enum):
     MAPPING = {
         NONE: [],
         BASENAME: ['--match-basename'],
-        EXTENSION: ['--match-with-extension'],
-        WITHOUT_EXTENSION: ['--match-without-extension']
+        EXTENSION: ['--match-extension'],
+        WITHOUT_EXTENSION: ['--no-match-extension']
```

Comment:
Commit 89dc9015bda754b37cc12c06c9af7919690a2fe6 renamed only --match-with-extension and --no-match-with-extension. --match-without-extension is still a perfectly valid option, which only considers files to be duplicates if the part before the extension matches.